versions.json

```diff
@@ -1,3 +1,4 @@
 [
+  "2.0.0",
```

Comment:
Shouldn't this be `2.1.0`?

Reply:
No, These are the old 2 versions in the document, i,e 2.0.0 & 1.0.0. 
This will auto created when we run a code this code - npm run docusaurus docs:version 2.0.0

From docusaurus doc >
"Tagging a new version"
First, make sure the current docs version (the ./docs directory) is ready to be frozen.
Enter a new version number.

npm run docusaurus docs:version 1.1.0

When tagging a new version, the document versioning mechanism will:

Copy the full docs/ folder contents into a new versioned_docs/version-[versionName]/ folder.
Create a versioned sidebars file based from your current [sidebar](https://docusaurus.io/docs/docs-introduction#sidebar) configuration (if it exists) - saved as versioned_sidebars/version-[versionName]-sidebars.json.
Append the new version number to versions.json.